src/platforms/android/androidvpnactivity.h

```diff
@@ -73,6 +73,7 @@ enum ServiceEvents {
   // permissions, to show the "you're connected" messages.
   EVENT_REQUEST_NOTIFICATION_PERMISSION = 8,
   EVENT_ONBOARDING_COMPLETED = 9,
+  EVENT_ALLOW_VPN_CONFIG_OPTION_SELECTED = 10,
```

Comment:
Consider if a name like EVENT_TUNNEL_CONFIGURATION_COMPLETED is better. Also, should a comment be added above this enum value?

Reply:
hmm, do you think that naming is still appropriate even though the configuration could be rejected?

Reply:
Sorry, please ignore that. Perhaps EVENT_VPN_CONFIG_PERMISSION_PROVIDED is better?